<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Websocket客户端--测试版</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>49</y>
     <width>511</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>消息记录列表</string>
   </property>
   <widget class="QTextEdit" name="textEdit_MsgList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_SendData">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>230</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器地址：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ServerAddress">
      <property name="text">
       <string>ws://192.168.1.31:8899</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>服务器名称：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ServerName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Connect">
      <property name="text">
       <string>连接服务器</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>307</y>
     <width>511</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>私聊(客户端名称)：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ClientName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_SendMsg">
      <property name="text">
       <string>发送消息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_ServerAddress</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_ServerName</zorder>
  <zorder>pushButton_Connect</zorder>
  <zorder>textEdit_SendData</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_ClientName</zorder>
  <zorder>pushButton_SendMsg</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
